core-bundle/contao/forms/Form.php

```diff
@@ -310,7 +310,7 @@ protected function compile()
 					$file->delete();
 				}
 
-				if (is_file($upload['tmp_name']))
+				if (!empty($upload['tmp_name']) && is_file($upload['tmp_name']))
```

Comment:
```suggestion
				if (is_file($upload['tmp_name'] ?? null))
```

Reply:
`is_file()` does not allow null as parameter I think.